spec/integration/network/http/api/indirected_routes_spec.rb

```diff
@@ -2,6 +2,8 @@
 require 'spec_helper'
 require 'puppet/network/http'
 require 'puppet/network/http/api/indirected_routes'
+require 'rack/mock' if Puppet.features.rack?
```

Comment:
not really. it's more idiomatic to keep them all together.

although it might make sense to move the conditional require to the end of the list.